doc/source/user_guide/legacy/scheme.rst

```diff
@@ -14,17 +14,17 @@ Examples
 .. code-block:: python
 
    >>> from ansys.fluent.core.services.scheme_eval import Symbol as S
-   >>> session.scheme_eval.eval([S('+'), 2, 3])
+   >>> session.scheme._eval([S('+'), 2, 3])
```

Comment:
Whether to document "_eval" now? It is private.

Reply:
Do not document it, thanks.

Reply:
We do not document private members generally but we can decide.

Reply:
Done, Thank you.